lms/resources/_js_config.py

```diff
@@ -253,6 +277,7 @@ def maybe_set_focused_user(self):
         In theory, though, the focused_user param could work outside of Canvas
         as well if we ever want it to.
 
+        :raise ConsumerKeyError: if request.lti_user.consumer_key isn't in the DB
```

Comment:
This could raise `ConsumerKeyError` before and it still can now, just added it to the docstring